test/cookbooks/acme_test/recipes/acme_dns_client.rb

```diff
@@ -0,0 +1,110 @@
+node.default['osl-acme']['pebble']['always_valid'] = false
+node.default['osl-acme']['pebble']['command'] = '/usr/local/bin/pebble -config /opt/pebble/test/config/pebble-config.json -dnsserver :8053'
+
+build_essential 'osl-acme-test' do
+  compile_time true
+end
+
+chef_gem 'bcrypt' do
+  compile_time true
+end
+
+#
+# Configure Bind
+#
+
+bind_service 'default' do
+  action [:create, :start]
+end
+
+cookbook_file '/etc/named.conf'
+
+directory '/var/named/master' do
+  recursive true
+end
+
+template '/var/named/master/db.example.org' do
+  source 'db.example.org.erb'
+  notifies :restart, 'bind_service[default]', :immediately
+end
+
+#
+# Setup PostgreSQL server
+#
+
+include_recipe 'osl-postgresql::server'
+
+db_config = data_bag_item('osl_acme', 'database')
+
+postgresql_database db_config['dbname']
+
+postgresql_user db_config['user'] do
+  database db_config['dbname']
+  password db_config['pass']
+  action :create
+end
+
+file '/var/lib/pgsql/12/data/pg_hba.conf' do
+  content <<-EOF
+local   all             all                                     trust
+host    all             all             0.0.0.0/0               trust
+  EOF
+  notifies :reload, 'service[postgresql]', :immediately
+end
+
+#
+# Configure acme-dns
+#
+
+osl_fakenic 'lo' do
```

Comment:
Please do create a fake nic on `lo`. Just create a new `eth1 interface for testing.